apps/pipeline/mix.exs

```diff
@@ -0,0 +1,42 @@
+defmodule Pipeline.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :pipeline,
+      version: "0.1.0",
+      build_path: "../../_build",
+      config_path: "../../config/config.exs",
+      deps_path: "../../deps",
+      lockfile: "../../mix.lock",
+      elixir: "~> 1.8",
+      start_permanent: Mix.env() == :prod,
+      test_paths: Mix.env() |> test_paths(),
+      deps: deps()
+    ]
+  end
+
+  def application do
+    [
+      extra_applications: [:logger],
+      mod: {Pipeline.Application, []}
+    ]
+  end
+
+  defp deps do
+    [
+      {:smart_city, "~> 3.0", override: true},
+      {:elsa, "~> 0.10.0"},
```

Comment:
update to latest

Reply:
Is it going to break everything like my last update did? Because I'd rather do that in another PR.